src/main/BrowserWinHandler.js

```diff
@@ -36,67 +36,84 @@ export default class BrowserWinHandler {
   }
 
   _create() {
-    // In order to get display size to match in windows: https://stackoverflow.com/questions/59385237/electron-window-dimensions-vs-screen-resolution
-    const scale_factor = screen.getPrimaryDisplay().scaleFactor;
-    console.log("Screen size scale factor: " + scale_factor); // allow-log
-    const { width, height } = screen.getPrimaryDisplay().workAreaSize;
-    console.log("Sceen work area width " + width + " height " + height); // allow-log
-    this.options.height = this.options.height / scale_factor;
-    this.options.width = this.options.width / scale_factor;
-    this.options.webPreferences.zoomFactor = this.options.webPreferences.zoomFactor / scale_factor;
+      // In order to get display size to match in windows: https://stackoverflow.com/questions/59385237/electron-window-dimensions-vs-screen-resolution
```

Comment:
I think you'll want to fix the indentation here or one of the pre-commit checks will fail. You can run pre-commit outside of an actual git commit with `pipenv run pre-commit run -a`